getting-started/vs-integration/new-project-wizard.md

```diff
@@ -77,20 +77,10 @@ In this screen, you can choose:
 
 ### Step 2: Select Project Template
 
-This step allows you to select one of the [predefined Telerik project](#predefined-project-templates) templates. The available templates vary depending on the selected target framework and hosting model in [**Step 1**](#step-1-select-initial-project-configuration).
+The **Create New Project** wizard provides a project template that is a blank solution based on the Microsoft template with added Telerik UI components (NuGet package reference, assets, service, and the `TelerikRootComponent`).
```

Comment:
"Blank solution" can be confusing, because it duplicates the name of an existing Visual Studio template that is truly blank. Moreover, our new template won't be blank - it will include a Drawer, Grid and some text information.

```suggestion
The **Create New Project** wizard generates a project that is based on the Microsoft Blazor template. The app includes some Telerik UI components and the [required NuGet package reference, static assets, localization service, and the `TelerikRootComponent`]({%slug getting-started/what-you-need%}).
```